### DISCUSSION TITLE
Counter-strike executable not found

### DISCUSSION THREAD

**Discussion by zsk1d** [akiver/cs-demo-manager#683] [Q&A] [answered]
I cant run cs2 from csdm prompt Counter strike executable not found

**Comment by zsk1d** [comment, marked as answer]
i fixed  
When you have two file paths of cs, csdm cannot successfully start cs.
Pls add a custom cs path setting.

**Reply by akiver** [reply]
See https://cs-demo-manager.com/docs/faq/playback#counter-strike-executable-not-found

**Reply by zsk1d** [reply]
thanks!
